Extension/src/LanguageServer/configurations.ts

```diff
@@ -1598,7 +1601,7 @@ export class CppProperties {
             }
 
             // Get compiler path without arguments before checking if it exists
-            resolvedCompilerPath = compilerPathAndArgs.compilerPath;
+            resolvedCompilerPath = compilerPathAndArgs.compilerPath ?? undefined;
```

Comment:
Same here, adding `| null` to the type of resolvedCompilerPath allows the `?? undefined` to be removed.